unit_tests/testLinearSolver_Multigrid/testLinearSolver_Multigrid.f90

```diff
@@ -0,0 +1,1031 @@
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
```

Comment:
Does updatedA get its own test?

Reply:
updatedA is tested in "testSoftReset"; the only important part of the updatedA routine in LS_MG.f90 is the call to the softreset function in pcmg_supplement.c.  The parent updatedA is tested in testUpdatedA of testLinearSolverTypes.f90